StripeIdentity/StripeIdentity/Source/Analytics/IdentityAnalyticsClient.swift

```diff
@@ -146,8 +148,29 @@ final class IdentityAnalyticsClient {
 
     private func logAnalytic(
         _ eventName: EventName,
-        metadata: [String: Any]
+        metadata: [String: Any],
+        verificationPage: StripeAPI.VerificationPage?
     ) {
+        if let verificationPage = verificationPage {
+            let userSessionId = verificationPage.userSessionId
+            let experiments = verificationPage.experiments
+
+            for exp in experiments {
```

Comment:
If if the experiment's eventName and metadatas matches the analytics event, then log an additional experimentExposure